### ISSUE TITLE
Internet disconnected message

### ISSUE THREAD

**Issue by Furqankhanzada** [AppBakerZ/FinanceBakerZ#63]

**Comment by Furqankhanzada** [comment]
@kashifsulaiman is it working ? i am unable to produce it.

**Comment by kashifsulaiman** [comment]
@Furqankhanzada Actually i hope you're trying this with disconnecting the internet. It won't reproduce in this way on local because server is also local. So It will work when we disconnect or stop `meteor server` on local. And it'll surely work on internet disconnect when it'll deploy.

**Comment by Furqankhanzada** [comment]
@kashifsulaiman move it to top. also can we convert it to Component ?

**Comment by kashifsulaiman** [comment]
@Furqankhanzada Yes, component would be a better way. I'll do it if I get free from knotable otherwise ask @SyedAsherAhmed to do so.

**Comment by Furqankhanzada** [comment]
@kashifsulaiman Good. i have asked to designer to design this error so work when you have design and free time.

**Comment by Furqankhanzada** [comment]
![connectionlostscreen](https://cloud.githubusercontent.com/assets/2400215/24139825/79edfe22-0e3f-11e7-9e0a-bc9346072214.jpg)

**Comment by Furqankhanzada** [comment]
@kashifsulaiman here is the design. So if it's better to create it without snackbar its fine. but if we can this with snackbar easily then try that first.

**Comment by Furqankhanzada** [comment]
@SyedAsherAhmed pul and check that is it working as expected ?

**Comment by SyedAsherAhmed** [comment]
@Furqankhanzada 
yes sir it is working as expected.I am adding icon to this snackbar.
